docs/source/visualisation.rst

```diff
@@ -7,7 +7,7 @@ Visualising the results of simulations
 
 Having run a simulation, it is likely that we will want to look at the
 results.  To do this, Firedrake supports saving data in VTK_
-format, suitable for visualisation in Paraview_ (amongst others).
```

Comment:
@colinjcotter is going to be very unhappy with you removing his "amongst".

Reply:
did I write that? I don't remember doing it

Reply:
"amongst" is your style...

Reply:
I blame ispell. However, I kept visualisation for some reason so we might as well be consistent.